src/systems/ec/implementations/vanilla/packages/ec-component-form-checkbox/ec-component-form-checkbox.scss

```diff
@@ -0,0 +1,91 @@
+/**
+ * Checkbox component
+ * @define checkbox
+ */
+
+// Import base
+@import '@ecl/ec-base/ec-base';
+
+@mixin ecl-checkbox(
+  $thickness-base: 1px,
+  $thickness-max: 3px,
+  $border-color: $ecl-color-grey-25,
+  $border-color-focus: $ecl-color-yellow-100,
+  $border-color-checked: $ecl-color-blue-100,
+  $fake-checkbox-size: 0.875rem
+) {
+  .ecl-checkbox {
+    align-items: center;
+    display: flex;
+    position: relative;
+  }
+
+  .ecl-checkbox__input {
+    cursor: pointer;
```

Comment:
why is there a `cursor: pointer` here? maybe it's inherited but we should not use a pointer cursor on checkboxes (the native checkbox html element uses the default cursor) :)

Reply:
btw, opacity is set to 0, there's no need to use a custom cursor

Reply:
in the end, `.ecl-checkbox__input` should be the equivalent of a "visually hidden" component but you'll need to tweak the markup a bit: the <label> should encapsulate the <input>, the <span>s for the icon and the text label. https://www.w3schools.com/howto/howto_css_custom_checkbox.asp

Reply:
@yhuard, i honestly don't like the markup proposed by the w3c in this example, ok that is the w3c but still the reason why a label would contain an input looks cumbersome to me.
I mean that the solution always existed to circumvent the need for attributes, id for the "input" and "for" for the label, but we are saying that we consider the label a mandatory element and therefore the "for" attribute for it, as well.
Why do you think we would have to go this way, instead?

Reply:
Actually, whether you put the `<input>` inside the `<label>` or not doesn't really matter (the `<input>` is visually hidden anyway). If the `<label>` encapsulates the `<input>`, the markup is lighter: no need for a `id` on the `<input>` and `for` on `<label>`. That's all. There's no strong need to go for one solution or the other, both are valid ;) 

But the label should at least contain the icon and the text (whether the text is visually hidden or not). Thus when you click on the icon, which is inside the label, you trigger the right checkbox automatically